tests/.achecker.yml

```diff
@@ -0,0 +1,66 @@
+# optional - Specify the rule archive
+# Default: latest
+# Run `npx achecker archives` for a list of valid ruleArchive ids and policy ids
+ruleArchive: latest
+
+# optional - Specify one or many policies to scan.
+# i.e. For one policy use policies: IBM_Accessibility
+# i.e. Multiple policies: IBM_Accessibility,WCAG_2_1
+# Run `npx achecker archives` for a list of valid ruleArchive ids and policy ids
+policies:
+    - IBM_Accessibility
+
+# optional - Specify one or many violation levels on which to fail the test
+#            i.e. If specified violation then the testcase will only fail if
+#                 a violation is found during the scan.
+# i.e. failLevels: violation
+# i.e. failLevels: violation,potential violation or refer to below as a list
+# Default: violation, potentialviolation
+failLevels:
+    - violation
+    - potentialviolation
+
+# optional - Specify one or many violation levels which should be reported
+#            i.e. If specified violation then in the report it would only contain
+#                 results which are level of violation.
+# i.e. reportLevels: violation
+# Valid values: violation, potentialviolation, recommendation, potentialrecommendation, manual
+# Default: violation, potentialviolation
+reportLevels:
+    - violation
+    - potentialviolation
+    - recommendation
+    - potentialrecommendation
+    - manual
+
+# Optional - In which formats should the results be output
+# Valid values: json, csv, xlsx, html, disable
+# Default: json
+outputFormat:
+    - json
+    - html
```

Comment:
Do you think we should provide a script to re-generate baselines ? 

I kept the `html` format so that users could quickly go through the accessibility issues using the browser as I demonstrated in the video.

Reply:
> Do you think we should provide a script to re-generate baselines ?

I think that's probably a good idea, would make sense to document in a section within the contributing docs also.

> I kept the html format so that users could quickly go through the accessibility issues using the browser as I demonstrated in the video.

Ok maybe we git ignore the html files, then we can still generate them easily but it keeps the change-set small. Similar to how we don't commit a coverage report for example :thinking: